VTEX - Logistics API.json

```diff
@@ -4410,7 +4412,7 @@
                     "Holidays"
                 ],
                 "summary": "Create or update holiday",
-                "description": "Creates or updates holidays through holiday ID.",
+                "description": "Create or update a [holiday](https://help.vtex.com/en/tutorial/registering-holidays--2ItOthSEAoyAmcwsuiO6Yk) using the holiday ID.",
```

Comment:
There are many endpoints whose description starts with a verb in 3rd person, like Creates ..., Retrieves..., etc. I would say it's almost a standard in our API reference. A few endpoints are exceptions to that.

I think using the 3rd person is also common externally, looking at other API reference portals ([Stripe](https://docs.stripe.com/api/balance_transactions/retrieve), [Shopify](https://shopify.dev/docs/api/admin-rest/2024-07/resources/customer#get-customers?ids=207119551,1073339474)).

Since you made these changes intentionally here, and I didn't find anything specific about this in our guidelines, I'd like to know the reason behind this choice. Do you think we should make it a standard to use the imperative / infinitive? If so, why?

Reply:
In case you decide to roll back to 3rd person, please update all endpoint descriptions where this change was applied in this PR

Reply:
I am changing the API descriptions for a more client-centric text to write about what the user can do instead of how the feature works, and I plan to change all endpoints. I do believe it would be beneficial for our guidelines to have this perspective in mind.